nodes/ceos/ceos.cfg

```diff
@@ -7,20 +7,28 @@ username admin privilege 15 secret admin
 !
 service routing protocols model multi-agent
 !
-{{ if .MgmtIPv4Gateway }}ip route 0.0.0.0/0 {{ .MgmtIPv4Gateway }}{{end}}
-{{ if .MgmtIPv6Gateway }}ipv6 route ::0/0 {{ .MgmtIPv6Gateway }}{{end}}
+{{- if .Env.MGMT_VRF }}
+vrf instance {{ .Env.MGMT_VRF }}
+!
+{{end}}
+{{ if .MgmtIPv4Gateway }}ip route {{ if .Env.MGMT_VRF }}vrf {{ .Env.MGMT_VRF }} {{end}}0.0.0.0/0 {{ .MgmtIPv4Gateway }}{{end}}
+{{ if .MgmtIPv6Gateway }}ipv6 route {{ if .Env.MGMT_VRF }}vrf {{ .Env.MGMT_VRF }} {{end}}::0/0 {{ .MgmtIPv6Gateway }}{{end}}
 !
 interface {{ .MgmtIntf }}
+{{ if .Env.MGMT_VRF }} vrf {{ .Env.MGMT_VRF }}{{end}}
 {{ if .MgmtIPv4Address }}ip address {{ .MgmtIPv4Address }}/{{.MgmtIPv4PrefixLength}}{{end}}
 {{ if .MgmtIPv6Address }}ipv6 address {{ .MgmtIPv6Address }}/{{.MgmtIPv6PrefixLength}}{{end}}
 !
 management api gnmi
    transport grpc default
+{{ if .Env.MGMT_VRF }} vrf {{ .Env.MGMT_VRF }}{{end}}
```

Comment:
misaligned spacing in the term `vrf` doesn't cause issues with eos? I think no, but just checking

Reply:
It didn't in my testing, but in the interest of completeness I adjusted the spacing.